recipes/input4mips-validation/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "input4mips-validation" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/input4mips_validation-{{ version }}.tar.gz
+  sha256: e1e31e0279621023bd0e5468eb02ba89661a9a0b685cd4b46f5dfa54634b121d
+
+build:
+  entry_points:
+    - input4mips-validation = input4mips_validation.cli:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.9.0,<4.0.0
+    - loguru >=0.7
+    - cf_xarray >=0.8
+    - xarray >=2022.0
+    - pint-xarray >=0.3
+    - numpy >=1.23,<2.0
+    - cftime >=1.6
+    - typing-extensions >=4.0
+    - pooch >=1.0
+    - netcdf4 >=1.0
+    - attrs >=22.0
+    - typer >=0.9.0,<0.10.0
+    - validators >=0.28.3,<0.29.0
+    - cattrs >=23.2.3,<24.0.0
```

Comment:
Are these strict upper bounds necessary or just "out of caution". If the latter, they are not usually needed and we prefer being optimistic in the honor of flexibility. If something does break, we can repodata patch and the constraints after releasing it without rebuilding anything.

Reply:
Ah yes these can all be removed. I think they're the result of using Grayskull to build the recipe based on a repository which uses poetry (which automatically adds these upper bounds). I'll remove them, the flexibility is definitely the goal (particular given there is also a 'locked' version).

Reply:
Removed upper bounds